### PR TITLE
Feature TMC#98 internal procedures

### PR DESCRIPTION
**Pull request by Lurkerpas**

### CODE REVIEW

File: sdl2promela/sdl/model.py [hunk 1]
```diff
@@ -409,6 +430,7 @@ def __init__(self):
         self.iteratorName = None
 
 
+# TODO Investigate whether the parameters should actually be Expressions
```
Comment (rphh) [inline sdl2promela/sdl/model.py:433]: TODO?

Comment (Lurkerpas) [reply sdl2promela/sdl/model.py:433]: Yes, TODO. This is not assumed to be changed in the scope of this task.

File: tests/unit/translator/resources/join.pml [hunk 2]
```diff
@@ -36,6 +36,6 @@ inline Complexsdl_0_PI_0_impulse(tmp_param_in)
   ::(global_state.complexsdl.state == Complexsdl_States_wait)->
     Complexsdl_0_transition(1);
   ::else->
-    break;
+    skip;
```
Comment (rphh) [inline tests/unit/translator/resources/join.pml:39]: Please explain this change

Comment (Lurkerpas) [reply tests/unit/translator/resources/join.pml:39]: break exits from a loop; skip is the proper nop statement

**Pull request merged**